### PR TITLE
RNs-4.13-stock:updates template file for 4.13 RNs

### PR DESCRIPTION
**Pull request by JoeAldinger**
RNS-4.13: updates the template file for 4.13 RNs assembly

Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-4837

Link to docs preview:
[RNs Preview](https://joealdinger.github.io/openshift-docs/RNs-4.13-stock/release_notes/ocp-4-13-release-notes.html)

QE review:
- [ ] QE has approved this change.

Additional information:
This was peer-reviewed in #54889. Things got weird so I started over.

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,1151 @@
...
+
+[id="ocp-4-13-olm"]
+=== Operator lifecycle
+
+[id="ocp-4-13-osdk"]
+=== Operator development
+
+[id="ocp-4-13-machine-api"]
+=== Machine API
+
```
Comment (stevsmit) [inline release_notes/ocp-4-13-release-notes.adoc:82]: Extra space here

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,1151 @@
...
+{product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title}, you can deploy OpenShift clusters to either on-premises or cloud environments.
+
+// Double check OP system versions
+{product-title} {product-version} is supported on {op-system-base-full} 8.4 and 8.5, as well as on {op-system-first} 4.13.
+
+You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
+//Removed the note per https://issues.redhat.com/browse/GRPA-3517
+
+//TODO: Remove this for 4.13
```
Comment (stevsmit) [inline release_notes/ocp-4-13-release-notes.adoc:27]: Don't forget to address this whenever appropriate.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,1151 @@
...
+==== {op-system-first}
+
+[discrete]
+[id="ocp-4-13-scalability-and-performance-bug-fixes"]
+==== Scalability and performance
+
+[discrete]
+[id="ocp-4-13-storage-bug-fixes"]
+==== Storage
+
```
Comment (stevsmit) [inline release_notes/ocp-4-13-release-notes.adoc:454]: Extra space

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,1151 @@
...
+|====
+|Feature |4.11 |4.12 | 4.13
+
+|Hosted control planes for {product-title} on bare metal
+|Not Available
+|Technology Preview
+|Technology Preview
+
+|Hosted control planes for {product-title} on Amazon Web Services (AWS)
+||Not Available
```
Comment (stevsmit) [inline release_notes/ocp-4-13-release-notes.adoc:947]: This line needs fixed as there are two bars |

File: release_notes/ocp-4-13-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,1151 @@
...
+
+|====
+
+[discrete]
+=== Networking deprecated and removed features
+
+.Networking deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.10 |4.11 | 4.12
```
Comment (stevsmit) [inline release_notes/ocp-4-13-release-notes.adoc:309]: This one still goes from 4.10-4.12

**Comment by stevsmit** [comment]
Looks good overall. A few things to fix.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-13-release-notes.adoc:27]: Will do

**Pull request merged**